spec/requests/form_templates_controller_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+RSpec.describe FormTemplatesController do
+  fab!(:user) { Fabricate(:user) }
+
+  before { SiteSetting.experimental_form_templates = true }
+
+  describe "#index" do
+    fab!(:form_template) { Fabricate(:form_template) }
+
+    context "when logged in as a user" do
+      before { sign_in(user) }
+
+      it "should return all forum templates" do
+        get "/form-templates.json"
+        expect(response.status).to eq(200)
+        json = response.parsed_body
+        expect(json["form_templates"]).to be_present
```

Comment:
Nitpick, but this assertion only checks that the endpoint returns something, it doesn't strictly check that it is returning "all templates" (in other words, we should create a few more form templates and ensure they're all output in the right order).